src/components/ThemeProvider/__stories__/ThemeProvider.stories.js

```diff
@@ -72,3 +76,36 @@ export const CustomClassSelector = {
   render: ThemeProviderCustomClassTemplate.bind({}),
   name: "Custom class selector"
 };
+
+export const WithSystemTheme = {
```

Comment:
I'm not sure that this example is clear enough. What do you want the user to learn from it?

I think that it should be set in two parts.
1. first - show a switch that activates the `systemTheme` mode with the light system theme (this should explain that the provider now controls the system theme).
2. second - showing a button that when clicking on it will toggle `light` and `dark` mode

Reply:
Made button a toggler itself, I think it will explain everything at once
- it's switching systemTheme on click
- Effect of theme change is displaying on the button itself